components/hab/src/main.rs

```diff
@@ -642,6 +642,9 @@ fn exec_subcommand_if_called(ui: &mut UI) -> Result<()> {
         ("pkg", "export", "cf") => {
             command::pkg::export::cf::start(ui, env::args_os().skip(4).collect())
         }
+        ("pkg", "export", "kubernetes") => {
```

Comment:
As stated in commit message - this won't work with `hab package export kubernetes`. Also, I was considering adding a shortcut like `hab pkg export k8s`. What do you think?